src/maths.rs

```diff
@@ -93,26 +98,33 @@ pub trait MathematicalOps {
     /// The Cumulative distribution function for a Normal distribution
     fn norm_cdf(&self) -> Decimal;
 
-    /// The Probability density function for a Normal distribution
+    /// The Probability density function for a Normal distribution.
     fn norm_pdf(&self) -> Decimal;
+
+    /// The Probability density function for a Normal distribution returning `None` on overflow.
+    fn checked_norm_pdf(&self) -> Option<Decimal>;
 }
 
 impl MathematicalOps for Decimal {
     fn exp(&self) -> Decimal {
+        self.exp_with_tolerance(EXP_TOLERANCE).unwrap()
```

Comment:
Ah.  OK. Easy to change.

Reply:
But not sure how useful the `should_panic` tests should be, since you're always internally calling a checked version which returns `None`.  If you control all the code, there is no uncontrollable panics and you don't really need those tests.

Reply:
Cool, no worries. I can add these later perhaps. The main reason I include them is to ensure all library outputs are asserted.